code/modules/mob/living/simple_animal/hostile/human/halloween.dm

```diff
@@ -0,0 +1,395 @@
+/*
+	Halloween related mobs
+*/
+
+/mob/living/simple_animal/hostile/humanoid/kitchen
+	faction = "kitchen_nightmares"
+	icon = 'icons/mob/hostile_humanoid.dmi'
+
+/mob/living/simple_animal/hostile/humanoid/kitchen/poutine
+	name = "poutine titan"
+	desc = "When hell came to Canada, Canada sent it right back with their own congealed conglomeration."
+
+	icon_state = "cheese_zombie"
+
+	health = 350
+	maxHealth = 350
+
+	move_to_delay = 20
+	speed = 5
+
+	melee_damage_lower = 20
+	melee_damage_upper = 35
+	attacktext = "smashes their cheese-covered gauntlets into"
+	attack_sound = 'sound/weapons/heavysmash.ogg'
+	corpse = null
+
+/mob/living/simple_animal/hostile/humanoid/kitchen/poutine/bullet_act(var/obj/item/projectile/P)
+	if(istype(P, /obj/item/projectile/beam))
+		P.damage /= rand(1.2,3)
```

Comment:
`rand` doesn't support non-natural numbers:

![](https://i.imgur.com/JVWCauI.png)

Reply:
You could do `rand(12,30)/10`.